docker-entrypoint.sh

```diff
@@ -267,6 +278,25 @@ docker_sql_escape_string_literal() {
 	echo "${escaped//\'/\\\'}"
 }
 
+# Creates replication user
+create_replica_user() {
+	if [[ "$MARIADB_REPLICATION_MODE" = "master" && -n "$MARIADB_REPLICATION_USER" ]]; then
```

Comment:
Single `[` as no requiring `[[` function is needed.

Reply:
Done 👍🏽